Source/WebCore/platform/graphics/transforms/TransformationMatrix.cpp

```diff
@@ -1168,7 +1283,7 @@ TransformationMatrix& TransformationMatrix::zoom(double zoomFactor)
 // this = mat * this.
 TransformationMatrix& TransformationMatrix::multiply(const TransformationMatrix& mat)
 {
-#if CPU(ARM64) && defined(_LP64)
+#if CPU(ARM64) && defined(ADDRESS64)
```

Comment:
This needs to be `CPU(ADDRESS64)`.

Reply:
Fixed in [088646b](https://github.com/WebKit/WebKit/commit/088646b8ecc9292b9e401fc9a950373f4948c3ad).